submissions/IRETIOGEDENGBE/HTML/Homepage.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
File names should be preferably in lowercase

Reply:
Page looks a bit distorted. 
![image](https://user-images.githubusercontent.com/11851535/69476619-6368cc00-0ddc-11ea-8505-e07e0d0ea7f7.png)